graphics/opencv4/Portfile

```
@@ -226,7 +226,13 @@ configure.args-append \
                     -DBUILD_opencv_python3:BOOL=OFF \
                     -DOPENCV_PYTHON_SKIP_DETECTION:BOOL=OFF \
                     -DPYTHON_EXECUTABLE:FILEPATH=/usr/bin/python \
-                    -DPYTHON_DEFAULT_EXECUTABLE:FILEPATH=/usr/bin/python
+                    -DPYTHON_DEFAULT_EXECUTABLE:FILEPATH=/usr/bin/python \
```

A backslash shouldn't be added at the end of this line.
```suggestion
                    -DPYTHON_DEFAULT_EXECUTABLE:FILEPATH=/usr/bin/python
```

---

Good catch!